x/ibc/05-port/types/port.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+import (
+	"strings"
+)
+
+// ValidatePortID validates that the provided port identifier is not empty
```

Comment:
More validation incoming here I would imagine...

Reply:
Up to the SDK, the spec intentionally does not fix this. I recommend a minimum length perhaps.

Reply:
Yeah, the spec didn't mention anything... although the uniqueness of the portID is validated when the port is retrieved from the store (i.e `port, found := k.GetPort(ctx, portID)` where `found == true`)